WasatchNET/Driver.cs

```diff
@@ -189,7 +197,10 @@ public int openAllSpectrometers()
             ////////////////////////////////////////////////////////////////////
 
             // Add 3rd-party USB spectrometers (e.g. Ocean Optics, etc)
-            if (deviceRegistries.Count > 0 && Environment.Is64BitProcess)
+            if (CheckLibrary("SeaBreeze"))
+                logger.debug("SeaBreeze appears installed");
+
+            if (deviceRegistries.Count > 0 && Environment.Is64BitProcess && CheckLibrary("SeaBreeze"))

```

Comment:
So you actually call LoadLibrary() twice?  I guess it would seem worth setting a boolean flag after the first check.

Reply:
Are there any problems if you LoadLibrary("SeaBreeze") on a 32-bit system?

Reply:
The first call is because it wasn't doing what I expected it to (the exact opposite in fact) and I just didn't remove it. The second is a great question, I'll check into it

Reply:
Not an issue if SB is NOT present. Let's see the other way

Reply:
Also not an issue if it is. LoadLibrary returns false even though the 64-bit version is installed

Reply:
I'll note that the first call should be removed in the next version. We've sent this to a customer so I want to keep it preserved as is